email_strides/ses.py

```diff
@@ -0,0 +1,251 @@
+import logging
+import os
+
+import boto3
+from botocore.config import Config as BotoConfig
+from botocore.exceptions import ClientError
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.DEBUG)
+
+# Use standard mode in order to retry on RequestLimitExceeded
+# and increase the default number of retries to 10
+ses_config = BotoConfig(
+    retries={
+        "mode": "standard",  # default mode is legacy
+        "max_attempts": 10,  # default for standard mode is 3
+    }
+)
+ses_client = boto3.client("ses", config=ses_config)
+
+
+def _table_row_style(i):
+    """
+    Alternating table row background colors
+    """
+    if i % 2 == 0:
+        return "style='background-color: WhiteSmoke;'"
+    else:
+        return ""
+
+
+def build_paragraph(text, html=False):
+    output = ""
+
+    if html:
+        # Put the paragraph in an invisible table to wrap long lines;
+        # give the table a single row with two cells, put the text in
+        # the first cell and let the second cell fill any extra space
+        output += (
+            "<table border='0' width='100%' "
+            "style='border-collapse: collapse;'><tr>"
+            f"<td width='600'>{text}</td><td></td>"
+            "</tr></table>"
+        )
+    else:
+        output += text + "\n"
+
+    return output
+
+
+def build_service_table(services, html=False):
```

Comment:
by "different format" I mean that the search results each need their own email templating based on the semantic meaning of the results.

and now that I'm looking back over this change, I see that I need much better comments